styles/style.css

```diff
@@ -8,13 +8,10 @@
 }
 
 .img-partners {
-    display:inline-block;  
-    margin:25px;
+    display: block;  
```

Comment:
this is the type of change I'd want to see on a test site first, will you please add one?